Pipfile

```diff
@@ -11,20 +11,10 @@ name = "pypi"
 [packages]
 pipenv = "*"
 requests = "*"
-
-[packages.aws-infrastructure]
```

Comment:
I'm guessing pipenv automatically did this when you installed autopep8?

Let's not do this destruction of my documentation?